receiver/kafkareceiver/config.go

```diff
@@ -60,6 +60,10 @@ type Config struct {
 
 	// Controls the way the messages are marked as consumed
 	MessageMarking MessageMarking `mapstructure:"message_marking"`
+
+	// Extract headers
+	HeaderExtraction bool     `mapstructure:"header_extraction"`
```

Comment:
Make sure you update the README to include the changes.

Reply:
Please also include the matching pattern which is `exact` since someone may want regex.

Reply:
Done in README